android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/LocationModule.java

```diff
@@ -154,6 +174,7 @@ private void stopWatching() {
     }
 
     SmartLocation.with(mScopedContext).location().stop();
+    SmartLocation.with(mScopedContext).geocoding().stop();
```

Comment:
ok, it's fine as long as it doesn't error if you call `.stop` when it hasn't been started